### ISSUE TITLE
Clarify dates at which software must be available for the available category

### ISSUE THREAD

**Issue by johntran-nv** [mlcommons/policies#83]
It looks like we were not clear on this, as I don't see it in the rules doc, but I believe the goal in the past was that software must be publicly available on submission day (not publication day).

**Comment by arjunsuresh** [comment]
yes, the rule says so (submission day). But in case of "closed software" I'm not sure how this availability can be determined.
